libbeat/idxmgmt/lifecycle/es_client_handler.go

```diff
@@ -95,7 +95,7 @@ func NewESClientHandler(c ESClient, info beat.Info, cfg RawConfig) (*ESClientHan
 	// if the user has set both to different values, throw a warning, as overwrite operations will probably fail
 	if c.IsServerless() {
 		if cfg.TemplateName != "" && cfg.TemplateName != name {
-			logp.L().Warnf("policy name is %s but template name is %s; under serverless, non-default template and policy names should be the same. Updates & overwrites may not work.")
+			logp.L().Warnf("policy name is %s but template name is %s; under serverless, non-default template and policy names should be the same. Updates & overwrites may not work.", name, cfg.TemplateName)
```

Comment:
Since this is serverless is policy name the DSL index pattern? And is template the index pattern for index creation and mappings?

Wondering if the language used here can be improved. Is policy even a concept is DSL?

Reply:
Yah, this came up in discussion of the initial PR, and I don't think we ever came to a conclusion before the PR was merged. The DSL name needs to match the template name in `setup.template.name`, since that variable is used to construct the data stream where the actual DSL policy is located. Technically we could remove the DSL policy name, but that means that the DSL setup becomes dependent on `setup.template.name` which might be unintuitive? Not sure.